src/client.ts

```diff
@@ -111,6 +112,28 @@ export class Client {
     }
   };
 
+  private _onKeyDown = (event: KeyboardEvent) => {
+    if (!this._environmentData.registeredKeys) {
+      return;
+    }
+
+    const keyData = Key.fromKeyEvent(event);
+    const shouldSend = this._environmentData.registeredKeys.some(
+      (key: string) => key === keyData.serialize()
+    );
+    if (!shouldSend) {
+      return;
+    }
+
+    this._sendToHost({
+      msgType: 'publish',
+      msg: {
+        topic: 'keydown.topic',
```

Comment:
This might be an abuse of the topic system so I can change to do something else

Reply:
I think i would prefer a direct message instead of the generic publish, since this is an Iframe event and not an app event.  It seems similar to  requestToast in that regards.